media/themes/Lautus/lautus.msstyles/textfiles/NormalNormal.INI

```diff
@@ -1019,35 +1019,32 @@ BgType = ImageFile
 FillColorHint = 36 94 220
 ImageCount = 1
 ImageFile = Normal\TaskbarBackground.bmp
-SizingMargins = 1, 0, 23, 0
+SizingMargins = 1, 0, 6, 2
 SizingType = Stretch
-TRANSPARENT = true
-TRANSPARENTCOLOR = 255 0 255
 
 [TaskBar.BackgroundLeft]
 BgType = ImageFile
 FillColorHint = 36 94 220
 ImageCount = 1
 ImageFile = Normal\TaskbarBackgroundVertical.bmp
-SizingMargins = 14, 3, 0, 0
+SizingMargins = 6, 2, 0, 0
 SizingType = Stretch
 
 [TaskBar.BackgroundRight]
 BgType = ImageFile
 FillColorHint = 36 94 220
 ImageCount = 1
 ImageFile = Normal\TaskbarBackgroundVertical.bmp
-SizingMargins = 14, 2, 0, 0
+SizingMargins = 6, 2, 0, 0
 SizingType = Stretch
 
 [TaskBar.BackgroundTop]
 BgType = ImageFile
 FillColorHint = 36 94 220
 ImageCount = 1
 ImageFile = Normal\TaskbarBackground.bmp
-SizingMargins = 1, 1, 15, 4
+SizingMargins = 1, 0, 6, 2
 SizingType = Stretch
-ContentMargins = 1, 1, 1, 1
```

Comment:
So you confirm that this ContentMargins is not wanted anymore here? (and in the other modified themes as well)

Reply:
Wasn't thinking, my bad